src/modalities/config/instantiation_models.py

```diff
@@ -32,7 +32,7 @@ class Training(BaseModel):
             global_training_log_interval_in_steps: Annotated[int, Field(strict=True, ge=1)]
             global_checkpointing_interval_in_steps: Annotated[int, Field(strict=True, ge=1)]
             global_evaluation_interval_in_steps: Annotated[int, Field(strict=True, ge=1)]
-            do_apply_activation_checkpointing: bool
+            activation_checkpointing_modules: Optional[List[str]]
```

Comment:
default value missing, as we discussed yesterday

Reply:
Done in fa1cca2ff1a4303e1b528641b61aa4d634679cac.